### PR TITLE
Add Piximedia adapter

### PR DESCRIPTION
**Pull request by christopher-allene-piximedia**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [X] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Add Piximedia adapter for Prebid 1.0.

This issue follows #2936 that has been previously closed (sorry, I was on extended vacation and returned online just a few hours after the issue was closed due to inactivity). The issue raised by @mike-chowla has been cleared, as mentioned.

Some test parameters for the bidder:

```     
    var adUnits = [{
      code: 'mpu',
      sizes: [[300, 250]],
      bids: [{
        bidder: 'piximedia',
        params: {
          siteId: 'PIXIMEDIA',
          placementId: 'PREBID'
        }
      }]
    }];

```

Our general contact email address is contact@piximedia.fr, and for anything related to this bidder, feel free to reach out to me at christopher.allene@piximedia.fr.

### CODE REVIEW

**Comment by jsnellbaker** [comment]
Hi @christopher-allene-piximedia 

Can you please add a docs-PR (repo [here](https://github.com/prebid/prebid.github.io)) to add your bidder info?  You can look at the other bidders [here](https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders) to get an idea - look at `AppNexus` or `Rubicon` if you want specific examples.

I'll take a look of your adapter code in the mean time.  Please let me know if you have any questions on the above.

File: test/spec/modules/piximediaBidAdapter_spec.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
+import { expect } from 'chai';
+import { spec } from 'modules/piximediaBidAdapter';
+
+describe('piximediaAdapterTest', () => {
```
[the remark this answers is not in the document]
Comment (christopher-allene-piximedia) [reply test/spec/modules/piximediaBidAdapter_spec.js]: Hi @jsnellbaker, I have changed all the arrow functions to standard functions.

**Comment by jsnellbaker** [comment]
Hi @christopher-allene-piximedia 

Thanks for making the updates to the `spec` file; that part looks good now.  Once you submit a docs PR - we should be good to merge.  If you can put that together today, we can include this PR in today's release.

**Comment by christopher-allene-piximedia** [comment]
Hi @jsnellbaker,

I have just submitted a PR for the documentation repo, updating the documentation we had for our pre-1.0 adapter.

Let me know if you need something else,
Christopher

**Pull request merged**